resources/web/wwi/X3dScene.js

```diff
@@ -173,17 +173,29 @@ export default class X3dScene {
         continue;
 
       let valid = true;
-      if (key === 'translation' && object instanceof WbTransform) {
+      if (key === 'translation') {
         const translation = convertStringToVec3(pose[key]);
 
-        if (typeof WbWorld.instance.viewpoint.followedId !== 'undefined' && WbWorld.instance.viewpoint.followedId === object.id) {
-          WbWorld.instance.viewpoint.setFollowedObjectDeltaPosition(translation, object.translation);
-          WbWorld.instance.viewpoint.updateFollowUp(time);
-        }
+        if (object instanceof WbTransform) {
+          if (typeof WbWorld.instance.viewpoint.followedId !== 'undefined' && WbWorld.instance.viewpoint.followedId === object.id) {
+            WbWorld.instance.viewpoint.setFollowedObjectDeltaPosition(translation, object.translation);
+            WbWorld.instance.viewpoint.updateFollowUp(time);
+          }
 
-        object.translation = translation;
-        if (WbWorld.instance.readyForUpdates)
-          object.applyTranslationToWren();
+          object.translation = translation;
+          if (WbWorld.instance.readyForUpdates)
+            object.applyTranslationToWren();
+        } else {
+          object.translation = translation;
+          if (WbWorld.instance.readyForUpdates) {
```

Comment:
Is not also better to check if `object` is an instance of WbTextureTransform just to avoid updating the appearance if other `translation` messages are received?